Extensions/DialogueTree/JsExtension.js

```diff
@@ -422,14 +486,14 @@ module.exports = {
         'JsPlatform/Extensions/yarn24.png',
         'JsPlatform/Extensions/yarn32.png'
       )
-      .addParameter('string', _('Command String'))
+      .addParameter('string', _('Command String'), '', false)
       .getCodeExtraInformation()
       .setFunctionName('gdjs.dialogueTree.isCommandCalled');
 
     extension
       .addCondition(
         'IsDialogueLineType',
-        _('The dialogue line type is'),
```

Comment:
Don't use partial sentences for titles of conditions. Put directly the name of what is being compared.

Reply:
(resolve once you've read this, I just want to highlight my changes ;))